test.js

```diff
@@ -93,10 +92,25 @@ test('parse', async (t) => {
         .data('settings', {fragment: false})
         .processSync('<title>Hi</title><h2>Hello world!')
         .toString(),
-      '<html><head><title>Hi</title></head><body><h2>Hello world!</h2></body></html>',
+      '<html xmlns="http://www.w3.org/1999/xhtml"><head><title>Hi</title></head><body><h2>Hello world!</h2></body></html>',
```

Comment:
I think this is what `jsdom` does: I don’t see it happening (`document.documentElement.outerHTML` printing an `xmlns`) in safari